edge-modules/azure-monitor/src/deployment.template.json

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
This file also doesn't seem useful since we already have ExampleDeployment.json.

Reply:
Removed.